projects/probational/Makefile

```diff
@@ -0,0 +1,14 @@
+GO_COMPILE=linuxkit/go-compile:3ba94f14de51b73551417e769d122815ec917ee7
```

Comment:
we're moving away from `go-compile` to using a base `alpine` image, example here: https://github.com/linuxkit/linuxkit/blob/master/pkg/sysctl/Dockerfile#L7